### PR TITLE
pythonPackages.django_accept_header: init at 0.3.2

### PR DESCRIPTION
**Pull request by lsix**
###### Things done
- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -9480,6 +9480,25 @@ in modules // {
...
+      sha256 = "1a8jwxx2y0vylpwhfp3j1zhdzx3d8y0pgc3566xg81md6091ngrl";
+    };
+
+    buildInputs =  with self; [ pytest ];
+    propagatedBuildInputs = with self; [ django ];
+
+    meta = {
+      descriotion = "A Django middleware that inspects the HTTP Accept headers sent by browsers";
+      homepage = https://github.com/fladi/django-accept-header;
+      license = lib.licenses.free; # ISC License (ISCL), MIT like
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: `stdenv.lib.licenses.free`, see travis

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (lsix) [inline pkgs/top-level/python-packages.nix]: Fixed.

Thanks

**Comment by FRidh** [comment]
are the tests run? Just asking because you added pytest.

**Review by kevincox — APPROVED** [review]

**Comment by lsix** [comment]
Sorry for late reply (quite busy those time)

Actually, the check phase do not find tests to run, but py.test it required as an install requirement… That is not the optimal solution since I should make sure that the tests are actually ran.

I’ll look into that, ASAP.

**Pull request closed without merging**